src/js/utils.js

```diff
@@ -0,0 +1,7 @@
+import { API_KEY, http } from "./constans";
+export const generatUrl = path =>`${http}${path}?api_key=${API_KEY}`;
+export const calculatePopularity = popularity => Math.round(popularity / 100)+'%';
+export const parseIsoDatetime= date =>{
+    let tmp = new Date(date).toString().split(' ');
```

Comment:
Use can you destruction here. `let [a,b,c] = new Date(date).toString().split(' ');`